storage/table.go

```diff
@@ -371,7 +375,7 @@ func extractContinuationTokenFromHeaders(h http.Header) *continuationToken {
 		NextRowKey:       h.Get(headerNextRowKey),
 	}
 
-	if ct.NextPartitionKey != "" && ct.NextRowKey != "" {
+	if ct.NextPartitionKey != "" || ct.NextRowKey != "" {
```

Comment:
Can it ever be the case that `NextRowKey` is not empty but `NextParitionKey` is?

Reply:
Storage docs didn't specifically mention if `NextParitionKey` could be empty, but it's a valid query to storage to have just `NextRowKey`.

Also from how other SDKs handle it, these three pagination headers `NextTableName`, `NextPartitionKey`, and `NextRowKey` don't have dependencies on each other. The non-empty ones will be added to the URL of the next request.